src/Microsoft.Extensions.Localization/ResourceLocationAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Localization
+{
+    public class ResourceLocationAttribute : Attribute
+    {
+        public ResourceLocationAttribute(string resourceFolder)
+        {
+            ResourceFolder = resourceFolder;
```

Comment:
Why we didn't throw `ArgumentNullException` instead of assuming the resource folder name is "Resources"